frontend/javascripts/dashboard/registration_form.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import { Form, Input, Button, Row, Col, Icon, Checkbox } from "antd";
+import React from "react";
+
+import { loginUser } from "admin/admin_rest_api";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import Request from "libs/request";
+import Store from "oxalis/throttled_store";
+import messages from "messages";
+
+const FormItem = Form.Item;
+const { Password } = Input;
+
+type Props = {|
+  form: Object,
+  onRegistered: true => void,
+|};
+
+type State = {
+  organizationName: string,
+};
+
+function generateOrganizationName() {
+  let output = "";
+  for (let i = 0; i < 8; i++) {
+    output += Math.floor(Math.random() * 255)
+      .toString(16)
+      .padStart(2, "0");
+  }
+  return output;
+}
+
+class RegistrationForm extends React.PureComponent<Props, State> {
```

Comment:
Do I understand this right, that this component is an adapted copy of the "old" `RegistrationForm`, right? Are the difference to big to unify these two components into one? If not, I'd rename this component to something like `AdminRegistrationForm` to avoid confusion with the other RegistrationForm.

Reply:
I think both forms have diverged a bit and it would make sense to split them into 2 components. Renaming is a good idea, tho.

Reply:
renamed to `SpotlightRegistrationForm` in latest commit.